pkg/cmd/roachtest/run.go

```diff
@@ -327,3 +333,102 @@ func testRunnerLogger(
 	shout(ctx, l, os.Stdout, "test runner logs in: %s", runnerLogPath)
 	return l, teeOpt
 }
+
+// runOperation sequentially runs one operation matched by the passed-in filter.
+func runOperation(register func(registry.Registry), filter string, clusterName string) error {
```

Comment:
What happens when the operation fails? I don't see any code related to handling that. Is it in a follow up PR?

Reply:
We just panic with errOperationFatal and have the process return an error, which is desirable.